ote/src/cljs/ote/app/controller/route.cljs

```diff
@@ -86,16 +163,22 @@
   AddStop
   (process-event [{feature :feature} app]
     ;; Add stop to current stop sequence
-    (-> app
-        (update-in [:route ::transit/stops]
-                   (fn [stop-sequence]
-                     (conj (or stop-sequence [])
-                           (merge (into {}
+    (let [current-stops (into [] (get-in app [:route ::transit/stops])) ;; Ensure, that we use vector and not list
```

Comment:
why is `::transit/stops` a list? can we fix it where it originates and remove this vector conversion